lib/tools/adb-commands.js

```diff
@@ -1338,6 +1338,21 @@ methods.getScreenSize = async function () {
   return null;
 };
 
+/**
+ * Get the current screen density in dpi
+ *
+ * @return {string} Device screen density as a string or _null_ if it
+ *                  cannot be determined
+ */
+methods.getScreenDensity = async function () {
+  let stdout = await this.shell(['wm', 'density']);
+  let size = new RegExp(/Physical density: ([^\r?\n]+)*/g).exec(stdout);
+  if (size && size.length >= 2) {
+    return size[1].trim();
+  }
+  return null;
```

Comment:
I'd rather throw an exception instead

Reply:
I'm matching previous functionality there from getScreenSize.  Since I intend on dropping this into the caps of the device for reading, I figure null makes sense?